SeleniumDemoQA/NUnitHomeworks/EnumHomework.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using NUnit.Framework;
+using static System.Formats.Asn1.AsnWriter;
+
+namespace NUnitHomeworks
+{
+    // TODO: modify enum so CheckCustomIntNumbersForTestDataAgeEnum pass
+    public enum TestDataAge
+    {
+        Child = 7,
+        Teenager = 14,
+        Adult = 30
+    }
+
+    //TODO: uncomment and implement tests so all Assert pass. Use such LINQ as Any(), Count(), Contains()
+    [TestFixture]
+    public class EnumHomework
+    {
+        [Test]
+        public void CheckCustomIntNumbersForTestDataAgeEnum()
+        {
+
+            Assert.That((int)TestDataAge.Child, Is.EqualTo(7));
+            Assert.That((int)TestDataAge.Teenager, Is.EqualTo(14));
+            Assert.That((int)TestDataAge.Adult, Is.EqualTo(30));
+        }
+
+        [Test]
+        public void SomeIntCorrespondsToSomeTestDataAgeValue()
+        {
+            var listOfInt = new List<int>() { 5, 14, 15 };
+
+            var isAnyIntCorrespondsToTestDataAge = listOfInt
+            .Any(age => new[] { (int)TestDataAge.Child, (int)TestDataAge.Teenager, (int)TestDataAge.Adult }.Contains(age));
```

Comment:
if Enum is extended with new values, this will not work or it need to be changed in every test. We need to get all int equivalents from Enum without manually copy/pasting them. For example:
var listOfEnumIntValues = Enum.GetValues(typeof(TestDataAge)).Cast<int>().ToList();

Reply:
updated according to notes